components/Buttons/CancelButton.js

```diff
@@ -0,0 +1,5 @@
+import { RedButton } from "./styles";
+
+export default function CancelButton() {
+  return <RedButton href="/">Abbrechen</RedButton>;
```

Comment:
If this is the only functionality of this button, you could use `Link` instead. It's works better for accessibility and search engine optimization